ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -948,7 +948,7 @@ def add_dbt_test_result(self, dbt_test: dict):
                     test_case_fqn = fqn.build(
                         self.metadata,
                         entity_type=TestCase,
-                        service_name=self.config.serviceName,
+                        service_name="*",
```

Comment:
Here let's use the `source_elements[0]` instead of `*`

Reply:
Ok

Reply:
Should we remove `self.config.serviceName` param as it is not needed anymore

Reply:
Yes if it is not used anywhere we can remove it

Reply:
Done 👍